doc/en/components/charts/chart-features-synchronization.md

```diff
@@ -0,0 +1,116 @@
+
+If you want to synchronize a set of charts, you can assign them the same name to the `SyncChannel` property and then specify whether or not to synchronize the charts horizontally and/or vertically by setting the `SynchronizeHorizontally` and `SynchronizeVertically` properties to the corresponding boolean value.
+
+Note that in order to synchronize either vertically and/or horizontally, you will need to set the `IsHorizontalZoomEnabled` and/or `IsVerticalZoomEnabled` property to `true`, respectively. A synchronized chart that is dependent on another chart will still zoom regardless of this property setting.
+
+## Code Snippet
+
+The following code snippet demonstrates how to synchronize a pair of chart controls in the $ProductName$ data chart:
+
+```razor
+<DataChart Height="600px" 
+    Width="100%"
+    IsHorizontalZoomEnabled="true"
+    IsVerticalZoomEnabled="true"
+    SyncChannel="ChannelA"
+    SynchronizedHorizontally="true"
+    SynchronizedVertically="true">
+
+</DataChart>
+
+<DataChart Height="600px" 
+    Width="100%"
+    IsHorizontalZoomEnabled="true"
+    IsVerticalZoomEnabled="true"
+    SyncChannel="ChannelA"
+    SynchronizedHorizontally="true"
+    SynchronizedVertically="true">
+
+</DataChart>
+```
+
+```html
+<igx-data-chart width="600px"
+    height="400px"
+    [dataSource]="data"
+    isHorizontalZoomEnabled=true
+    isVerticalZoomEnabled=true
+    syncChannel="ChannelA"
+    synchronizeHorizontally=true
+    synchronizeVertically=true>
+</igx-data-chart>
+
+<igx-data-chart width="600px"
+    height="400px"
+    [dataSource]="data"
+    isHorizontalZoomEnabled=true
+    isVerticalZoomEnabled=true
+    syncChannel="ChannelA"
+    synchronizeHorizontally=true
+    synchronizeVertically=true>
+</igx-data-chart>
+```
+
+```tsx
+<IgrDataChart width="600px"
+    height="400px"
+    dataSource={this.data}
+    isHorizontalZoomEnabled={true}
+    isVerticalZoomEnabled={true}
+    syncChannel="ChannelA"
+    synchronizeHorizontally={true}
+    synchronizeVertically={true} />
+
+
+<IgrDataChart width="600px"
+    height="400px"
+    dataSource={this.data}
+    isHorizontalZoomEnabled={true}
+    isVerticalZoomEnabled={true}
+    syncChannel="ChannelA"
+    synchronizeHorizontally={true}
+    synchronizeVertically={true} />
+```
+
+```html
+<igc-data-chart id="chart1"
+    width="600px"
+    height="400px"
+    is-horizontal-zoom-enabled="true"
+    is-vertical-zoom-enabled="true"
+    sync-channel="ChannelA"
+    synchronize-horizontally="true"
+    synchronize-vertically="true">
+</igc-data-chart>
+
+<igc-data-chart id="chart2"
+    width="600px"
+    height="400px"
+    is-horizontal-zoom-enabled="true"
+    is-vertical-zoom-enabled="true"
+    sync-channel="ChannelA"
+    synchronize-horizontally="true"
+    synchronize-vertically="true">
+</igc-data-chart>
+```
```

Comment:
all above code snippets should have different data binding and they should have IDs:


```html
<igx-data-chart width="600px" height="400px" id="chart1" [dataSource]="dataSouce1">
</igx-data-chart>

<igx-data-chart width="600px" height="400px" id="chart2" [dataSource]="dataSouce2">
</igx-data-chart>
```

Reply:
pending